apps/challenges/migrations/0067_add_sub_meta_att_schema_to_challenge_phase.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can we please change the filename to `0067_add_sub_meta_att_schema_to_challenge_phase.py` -> `0067_add_submission_meta_attribute_field_to_challenge_phase.py`